kettle-plugins/formats-meta/src/main/java/org/pentaho/big/data/kettle/plugins/formats/parquet/output/ParquetOutputMetaBase.java

```diff
@@ -395,11 +396,11 @@ public String getCompressionType() {
 
   public void setCompressionType( String value ) {
     compressionType =
-      StringUtil.isVariable( value ) ? value : parseFromToString( value, CompressionType.values(), CompressionType.NONE ).name();
+      StringUtil.isVariable( value ) ? value : parseFromToString( value, CompressionCodecName.values(), CompressionCodecName.UNCOMPRESSED ).name();
```

Comment:
Changing the compression type from NONE to UNCOMPRESSED.
Have you verified existing ktrs with compression type as NONE already set?

This ktr should be verified with changing the compression type.

You might need to run using PAN command as well